class-13-map-reduce/aliriza-ayshesaflo/map-exercise/index.js

```diff
@@ -0,0 +1,73 @@
+// First
+
+let randomString = "this is a random string";
+
+function capitalize(str) {
+  let words = str.split(' ');
+  //console.log(words);
+
+  let capitalisedWords = words.map(function(letter) {
+    return letter.toUpperCase();
+  })
+
+  let capitalisedString = capitalisedWords.join(" ");
+  console.log(capitalisedString)
+}
+
+// capitalize(randomString);
+
+
+// Second
+
+let randomString1 = "hey gurl, lets javascript together sometime"
+
+const swapCase = function(string) {
+  let words = string.split(" ");
+
+  let modifiedWords = [];
+
+  let i = 0;
+  while (i < words.length) {
```

Comment:
@ARS-coding this exercise is telling you to use map specifically, you don't need a while loop here, you should use map instead

Reply:
All done. :)